library/pom.xml

```diff
@@ -35,6 +35,7 @@
   <name>Apache Apex Malhar (incubating) Library</name>
 
   <properties>
+    <checkstyle.console>false</checkstyle.console>
```

Comment:
This is how it is in Apex core as well by @vrozov so was mimicking it.
I think the reason is that by default we don't want to print all the existing violations to console so we set the user property ```checkstyle.console``` to ```false```. However we give the user the option to see the existing violations by ```mvn checkstyle:check -Dcheckstyle.console```  if they want to see the existing violations.

If ```logViolationsToConsole``` is set to false, then user will not be able to spit out violations to the console.